platform/default/sqlite3.hpp

```diff
@@ -42,27 +48,23 @@ class Database {
 
     explicit operator bool() const;
 
-    static void errorLogCallback(void *arg, const int err, const char *msg);
     void setBusyTimeout(std::chrono::milliseconds);
     void exec(const std::string &sql);
     Statement prepare(const char *query);
 
-    int64_t lastInsertRowid() const;
```

Comment:
As methods on `Statement`, `lastInsertRowid` and `changes` invite misuse in this implementation:

```
Statement stmt1(...);
Statement stmt2(...);
stmt1.execute();
stmt2.execute();
stmt1.lastInsertRowid(); // Would actually return row ID from stmt2
```

We need to either keep them as `Database` methods, or cache the values in `Statement` after every execution so that code like the above behaves as expected. Also, we should validate that the Qt implementation do so as well.